Adyen/Model/MarketPay/Transaction.cs

```diff
@@ -448,8 +585,7 @@ public bool Equals(Transaction input)
                 ) &&
                 (
                     this.TransactionStatus == input.TransactionStatus ||
-                    (this.TransactionStatus != null &&
-                    this.TransactionStatus.Equals(input.TransactionStatus))
+                    this.TransactionStatus.Equals(input.TransactionStatus)
```

Comment:
is the `!= null` check not needed here? it seems to be used in all other statements

Reply:
An enum value cannot be null. It is a value type like an int.